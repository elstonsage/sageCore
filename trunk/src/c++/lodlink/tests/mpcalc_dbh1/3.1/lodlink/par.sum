1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LODLINK GENERAL PARAMETERS
 --------------------------
     TITLE ............................... DBH data.  Lodlink parameter file.                                              
     NUMBER OF MARKER LOCI ...............    3
     FEMALE AND MALE SYMBOLS ............. F  M
     MORTON'S HOMOGENEITY TEST ........... NO 
     SMITH'S HOMOGENEITY TEST AND TEST
     FOR LINKAGE UNDER HETEROGENEITY ..... NO 
     NUMBER OF PEDIGREES .................    4
     RECOMBINATION FRACTION 
      -- FOR MALES, (THETA M) ............   .0000    .0100    .0500    .1000    .2000    .3000    .4000
      -- FOR FEMALES, (THETA F) ..........   .0000    .0100    .0500    .1000    .2000    .3000    .4000
1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS AK1     
 ==========================================================

                            -------------------------------------------------------------------
 THETA M                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 THETA F                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 ----------------------------------------------------------------------------------------------
 !   PEDIGREE   NUMBER OF   !                                                                 !
 !   NUMBER     INDIVIDUALS ! LOD SCORES                                                      !
 ----------------------------------------------------------------------------------------------
 !   00001         93       !     .1573    .1517    .1302    .1050    .0613    .0280    .0071 !
 !   00003        395       ! -INFINITY  -1.1063   -.4608   -.2269   -.0595   -.0110   -.0007 !
 !   00005         88       !     .0385    .0370    .0314    .0250    .0142    .0064    .0016 !
 !   00009        347       !     .3150    .3058    .2691    .2238    .1383    .0661    .0170 !
 ----------------------------------------------------------------------------------------------
 !                923       ! -INFINITY   -.6117   -.0301    .1269    .1543    .0894    .0251 !
 ==============================================================================================
1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS ORM     
 ==========================================================

                            -------------------------------------------------------------------
 THETA M                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 THETA F                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 ----------------------------------------------------------------------------------------------
 !   PEDIGREE   NUMBER OF   !                                                                 !
 !   NUMBER     INDIVIDUALS ! LOD SCORES                                                      !
 ----------------------------------------------------------------------------------------------
 !   00001         93       ! -INFINITY  -6.5825  -3.1828  -1.8288   -.6895   -.2290   -.0486 !
 !   00003        395       ! -INFINITY -22.4778  -9.0410  -3.7438    .1019    .9042    .5375 !
 !   00005         88       ! -INFINITY  -2.8063  -1.3878   -.7977   -.2995   -.1119   -.0424 !
 !   00009        347       ! -INFINITY -16.1081  -7.0700  -3.6424  -1.0607   -.1762    .0497 !
 ----------------------------------------------------------------------------------------------
 !                923       ! -INFINITY -47.9749 -20.6816 -10.0127  -1.9478    .3871    .4962 !
 ==============================================================================================
