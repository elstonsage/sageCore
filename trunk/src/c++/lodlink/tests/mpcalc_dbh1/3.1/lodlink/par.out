1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LODLINK GENERAL PARAMETERS
 --------------------------
     TITLE ............................... DBH data.  Lodlink parameter file.                                              
     NUMBER OF MARKER LOCI ...............    3
     FEMALE AND MALE SYMBOLS ............. F  M
     MORTON'S HOMOGENEITY TEST ........... NO 
     SMITH'S HOMOGENEITY TEST AND TEST
     FOR LINKAGE UNDER HETEROGENEITY ..... NO 
     NUMBER OF PEDIGREES .................    4
     RECOMBINATION FRACTION 
      -- FOR MALES, (THETA M) ............   .0000    .0100    .0500    .1000    .2000    .3000    .4000
      -- FOR FEMALES, (THETA F) ..........   .0000    .0100    .0500    .1000    .2000    .3000    .4000

 Locus ABO     

 A1  =   .1904, A2  =   .0612, B   =   .0728, O   =   .6756, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   A1  A1       1                .036252
    2   A1  A2       1                .023305
    3   A1  B        2                .027722
    4   A1  O        1                .257268
    5   A2  A2       3                .003745
    6   A2  B        4                .008911
    7   A2  O        3                .082693
    8   B   B        5                .005300
    9   B   O        5                .098367
   10   O   O        6                .456435

 Locus AK1     

 1   =   .9620, 2   =   .0380, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   1   1        1                .925444
    2   1   2        2                .073112
    3   2   2        3                .001444

 Locus ORM     

 F   =   .6400, S   =   .3600, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   F   F        1                .409600
    2   F   S        2                .460800
    3   S   S        3                .129600

 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS AK1     
 ==========================================================


 FAMILY NUMBER    1 (ID. IS 00001)
 ---------------------------------

 %LODLINK-I: There are   35 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      -20.892970       .157261
      .0100    .0100      -20.898521       .151710
      .0500    .0500      -20.920021       .130210
      .1000    .1000      -20.945189       .105042
      .2000    .2000      -20.988941       .061290
      .3000    .3000      -21.022215       .028016
      .4000    .4000      -21.043102       .007129
      .5000    .5000      -21.050231       .000000

 FAMILY NUMBER    2 (ID. IS 00003)
 ---------------------------------

 %LODLINK-I: There are  120 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      ----------     -INFINITY
      .0100    .0100     -112.884754     -1.106256
      .0500    .0500     -112.239276      -.460778
      .1000    .1000     -112.005444      -.226946
      .2000    .2000     -111.838001      -.059503
      .3000    .3000     -111.789546      -.011048
      .4000    .4000     -111.779196      -.000698
      .5000    .5000     -111.778498       .000000

 FAMILY NUMBER    3 (ID. IS 00005)
 ---------------------------------

 %LODLINK-I: There are   30 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      -27.070432       .038506
      .0100    .0100      -27.071900       .037038
      .0500    .0500      -27.077520       .031419
      .1000    .1000      -27.083951       .024988
      .2000    .2000      -27.094726       .014212
      .3000    .3000      -27.102569       .006369
      .4000    .4000      -27.107338       .001601
      .5000    .5000      -27.108938       .000000

 FAMILY NUMBER    4 (ID. IS 00009)
 ---------------------------------

 %LODLINK-I: There are  139 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      -99.550376       .315028
      .0100    .0100      -99.559589       .305815
      .0500    .0500      -99.596314       .269090
      .1000    .1000      -99.641583       .223821
      .2000    .2000      -99.727124       .138280
      .3000    .3000      -99.799295       .066109
      .4000    .4000      -99.848363       .017041
      .5000    .5000      -99.865404       .000000

 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS ORM     
 ==========================================================


 FAMILY NUMBER    1 (ID. IS 00001)
 ---------------------------------

 %LODLINK-I: There are   36 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      ----------     -INFINITY
      .0100    .0100      -49.155874     -6.582535
      .0500    .0500      -45.756126     -3.182787
      .1000    .1000      -44.402116     -1.828777
      .2000    .2000      -43.262875      -.689536
      .3000    .3000      -42.802374      -.229035
      .4000    .4000      -42.621938      -.048599
      .5000    .5000      -42.573339       .000000

 FAMILY NUMBER    2 (ID. IS 00003)
 ---------------------------------

 %LODLINK-I: There are  120 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      ----------     -INFINITY
      .0100    .0100     -212.216560    -22.477840
      .0500    .0500     -198.779756     -9.041036
      .1000    .1000     -193.482552     -3.743831
      .2000    .2000     -189.636776       .101945
      .3000    .3000     -188.834494       .904226
      .4000    .4000     -189.201194       .537526
      .5000    .5000     -189.738720       .000000

 FAMILY NUMBER    3 (ID. IS 00005)
 ---------------------------------

 %LODLINK-I: There are   31 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      ----------     -INFINITY
      .0100    .0100      -40.985047     -2.806342
      .0500    .0500      -39.566491     -1.387786
      .1000    .1000      -38.976372      -.797667
      .2000    .2000      -38.478240      -.299534
      .3000    .3000      -38.290579      -.111873
      .4000    .4000      -38.221130      -.042424
      .5000    .5000      -38.178706       .000000

 FAMILY NUMBER    4 (ID. IS 00009)
 ---------------------------------

 %LODLINK-I: There are  139 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      ----------     -INFINITY
      .0100    .0100     -183.407876    -16.108136
      .0500    .0500     -174.369718     -7.069978
      .1000    .1000     -170.942178     -3.642437
      .2000    .2000     -168.360422     -1.060682
      .3000    .3000     -167.475939      -.176199
      .4000    .4000     -167.250087       .049654
      .5000    .5000     -167.299741       .000000
